include/seqan3/core/metafunction/iterator.hpp

```diff
@@ -113,6 +113,110 @@ struct size_type<it_t>
 
 // see specialisation for ranges in core/metafunction/range.hpp
 
+// ----------------------------------------------------------------------------
+// iterator_tag
+// ----------------------------------------------------------------------------
+
+/*!\brief Type metafunction that deduces the
+ * [iterator_category](https://en.cppreference.com/w/cpp/iterator/iterator_tags) from the modelled
+ * concept [Type metafunction].
+ * \tparam it_t The type to operate on.
+ *
+ * <table>
+ *   <tr>
+ *     <th>Modelled concept</th>
+ *     <th>iterator_tag<it_t>::type</th>
+ *   </tr>
+ *   <tr>
+ *     <td>\ref std::InputIterator "std::InputIterator<it_t>"</td>
```

Comment:
this works in a html table? I tried this as well but I was confused because it would just print the full string into the table

Reply:
<img width="1113" alt="Screen Shot 2019-03-28 at 22 08 59" src="https://user-images.githubusercontent.com/12967715/55193688-fbf47100-51a7-11e9-9ee1-fa69914ad4d0.png">